<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryWizard</class>
 <widget class="QDialog" name="EntryWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="LN_AM">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_AM">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>55</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="BTNBX_RES">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Reset</set>
   </property>
  </widget>
  <widget class="QComboBox" name="CMBX_ADD">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Income</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Expenses</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="LBL_ADD">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>365</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LN_DES">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>551</width>
     <height>61</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::InputMethodHint::ImhNone</set>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_DES">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMBX_MTH">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Monthly</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Quarterly</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yearly</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>One-time</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="LBL_ITV">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Interval:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SPBX_MTH">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
    background-color: white;
    border: 2px solid #bdc3c7;
    border-radius: 4px;
    padding: 2px 20px 2px 4px;
    min-height: 22px;
    font-size: 13px;
}

QSpinBox:focus {
    border-color: #3498db;
}

/* ✅ Up-Button */
QSpinBox::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right;
    width: 18px;
    height: 15px;
    border-left: 1px solid #bdc3c7;
    border-bottom: 1px solid #bdc3c7;
    border-top-right-radius: 2px;
    border-bottom-left-radius: 0px;
    background-color: #ecf0f1;
    image: url(:/img/chevron-up.png);
    image-position: center;
}

/* ✅ Down-Button */
QSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: bottom right;
    width: 18px;
    height: 15px;
    border-left: 1px solid #bdc3c7;
    border-top: 1px solid #bdc3c7;
    border-bottom-right-radius: 2px;
    border-top-left-radius: 0px;
    background-color: #ecf0f1;
    image: url(:/img/chevron-down.png);
    image-position: center;
}

/* ✅ Hover-Effekte */
QSpinBox::up-button:hover, QSpinBox::down-button:hover {
    background-color: #d5dbdb;
}

/* ✅ Gedrückt-Effekte */
QSpinBox::up-button:pressed, QSpinBox::down-button:pressed {
    background-color: #bdc3c7;
}

/* ✅ Deaktiviert-Effekte */
QSpinBox::up-button:disabled, QSpinBox::down-button:disabled {
    background-color: #ecf0f1;
    color: #bdc3c7;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="suffix">
    <string> Months</string>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::StepType::DefaultStepType</enum>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_EUR">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>55</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_ERR_TTL">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>45</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string>Invalid!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_ERR_EMT">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string>Amount can not be empty!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_ERR_ICR">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string>Incorrect format!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DEL">
   <property name="geometry">
    <rect>
     <x>519</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
